owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java

```diff
@@ -162,8 +161,10 @@ private void setBackgroundText() {
             if (!mSyncInProgress) {
                 // In case folder list is empty
                 message = R.string.file_list_empty_moving;
-                listFragment.getProgressBar().setVisibility(View.GONE);
-                listFragment.getShadowView().setVisibility(View.VISIBLE);
+                if (listFragment.getProgressBar() != null && listFragment.getShadowView() != null) {
```

Comment:
Is it possible that `listFragment.getProgressBar()` is not null while `listFragment.getShadowView()` is null and we are not properly setting the progressBar visibility in that case?

Reply:
I would say that it is not posible at all. Both are included in the same layout file (`top_progress_bar`) and are initialized at the same time.